wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -164,109 +139,119 @@ class ConversationSenderMessageDetailsCell: UIView, ConversationMessageCell {
+        var attributedString = NSMutableAttributedString(
+            string: fullName,
+            attributes: [
+                .foregroundColor: textColor,
+                .font: UIFont.mediumSemiboldFont]
+        )
 
-    private func configureAuthorLabel(user: UserType, fullName: String, textColor: UIColor) {
-        let textColor: UIColor
-        var attributedString = NSMutableAttributedString(string: fullName)
+        switch object.indicator {
 
-        if user.isServiceUser {
-            icon = .bot
-            textColor = SemanticColors.Label.textDefault
+        case .deleted:
             attributedString.append(
                 stringForAttachment(
-                    named: .bot,
-                    caption: fullName
+                    named: .trash,
+                    imageSize: 8
+                )
+            )
+
+        case .edited:
+            attributedString.append(
+                stringForAttachment(
+                    named: .pencil,
+                    imageSize: 8
+                )
+            )
+        case .none: break
+
+        }
+
+        switch object.teamRoleIndicator {
+
+        case .guest:
+            accessibilityIdentifier = "img.guest"
+            attributedString.append(
+                stringForAttachment(
+                    named: .guest,
+                    imageSize: 12
                 )
             )
-        } else if user.isExternalPartner {
-            icon = .externalPartner
-            textColor = user.accentColor
+        case .externalPartner:
+            accessibilityIdentifier = "img.externalPartner"
             attributedString.append(
                 stringForAttachment(
                     named: .externalPartner,
-                    caption: fullName
+                    imageSize: 12
+
                 )
             )
-        } else if user.isFederated {
-            icon = .federated
-            textColor = user.accentColor
+        case .federated:
+            accessibilityIdentifier = "img.federatedUser"
             attributedString.append(
                 stringForAttachment(
                     named: .federated,
-                    caption: fullName
+                    imageSize: 12
                 )
             )
-        } else if !user.isTeamMember,
-                  let selfUser = SelfUser.provider?.selfUser,
-                  selfUser.isTeamMember {
-            icon = .guest
-            textColor = user.accentColor
+        case .service:
+            accessibilityIdentifier = "img.serviceUser"
             attributedString.append(
                 stringForAttachment(
-                    named: .guest,
-                    caption: fullName
+                    named: .bot,
+                    imageSize: 12
                 )
             )
 
-        } else {
-            icon = .none
-
+        default:
+            accessibilityIdentifier = "img.member"
```

Comment:
Is this the accessibility identifier of the cell?

Reply:
Yes!